.rustfmt.toml

```diff
@@ -10,15 +10,46 @@ reorder_imports = true
 reorder_modules = true
 tab_spaces = 4
 use_field_init_shorthand = true
+use_small_heuristics = "Default"
 use_try_shorthand = true
-edition = "2018"
 
-# unstable options (for one-off fixes and CI)
-merge_imports = true
-wrap_comments = true
+# unstable options
+binop_separator = "Front"
+blank_lines_lower_bound = 0
```

Comment:
`0` is the default value
https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#blank_lines_lower_bound